service/history/shard/context_impl.go

```diff
@@ -1947,6 +1947,7 @@ func (s *ContextImpl) acquireShard() {
 
 	// keep retrying except ShardOwnershipLostError or lifecycle context ended
 	acquireShardRetryable := func(err error) bool {
+		s.contextTaggedLogger.Error("Error acquiring shard", tag.Error(err))
```

Comment:
would be useful to include if we are retrying or not. Could be done by do log in defer func and include return value of isRetryable  in the log.

Reply:
Done.